Gulpfile.ts

```diff
@@ -69,7 +69,7 @@ function exec(cmd: string, args: string[], complete: () => void = (() => {}), er
     console.log(`${cmd} ${args.join(" ")}`);
     // TODO (weswig): Update child_process types to add windowsVerbatimArguments to the type definition
     const subshellFlag = isWin ? "/c" : "-c";
-    const command = isWin ? [cmd, ...args] : [`${cmd} ${args.join(" ")}`];
+    const command = isWin ? [cmd.indexOf(" ") >= 0 ? `"${cmd}"` : cmd, ...args] : [`${cmd} ${args.join(" ")}`];

```

Comment:
Please break up the expressions here to convey intent a little better. I have no idea what's going on on this line.